packages/scandipwa/src/component/MyAccountAddressBook/MyAccountAddressBook.container.js

```diff
@@ -40,6 +40,15 @@ export class MyAccountAddressBookContainer extends PureComponent {
         showCreateNewPopup: this.showCreateNewPopup.bind(this)
     };
 
+    __construct(props) {
+        super.__construct(props);
+
+        const { changeTabName, tabMap } = props;
+        const { tabName } = tabMap[ADDRESS_BOOK];
+
+        changeTabName(tabName);
```

Comment:
Please don't make such actions inside the constructor.
The purpose of the constructor in React components is to initialize values.
Please perform all changes inside the lifecycle methods.